dotcom-rendering/src/server/handler.article.web.ts

```diff
@@ -13,7 +13,12 @@ export const handleArticle: RequestHandler = ({ body }, res) => {
 	recordTypeAndPlatform('article', 'web');
 
 	const frontendData = validateAsArticleType(body);
-	const article = enhanceArticleType(frontendData, 'Web');
+	let article = enhanceArticleType(frontendData, 'Web');
+
+	if (article.frontendData.crossword) {
+		article = enhanceCrosswordArticle(article);
```

Comment:
Is there a way we can run this as part of the other enhancers, i.e. within the `enhanceArticleType` function?

I think the net effect of this function is to append a new block to the article? Therefore it could perhaps live in the `enhanceBlocks` function? That's the place where modifications to the blocks are typically handled.

https://github.com/guardian/dotcom-rendering/blob/411e92829b8c96887e68f3d13b5d1c597aa617b5/dotcom-rendering/src/model/enhanceBlocks.ts#L98-L115

Alternatively, it could be concatenated onto the result of that function within the `enhanceArticleType` function, whenever a crossword is present.

https://github.com/guardian/dotcom-rendering/blob/411e92829b8c96887e68f3d13b5d1c597aa617b5/dotcom-rendering/src/types/article.ts#L43-L49

Reply:
currently it's being applied if `article.frontendData.crossword` is truthy, but iiuc, `enhanceBlocks` would need `format` to respond to, which we don't have yet, is that right?